qiskit/aqua/algorithms/eigen_solvers/__init__.py

```diff
@@ -12,11 +12,8 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-""" adaptive packages """
+""" Eigen Solvers Package """
```

Comment:
To clearly distinguish from min_eigen_solvers and also because this is closer to how EOM and other k-eigensolvers actually work, maybe we could make these "k_eigen_solvers" instead of just "eigensolvers"?